packages/metasploit-framework/build.sh

```diff
@@ -0,0 +1,40 @@
+TERMUX_PKG_HOMEPAGE=https://www.metasploit.com/
+TERMUX_PKG_DESCRIPTION="framework for pentesting"
+TERMUX_PKG_VERSION=4.16.2
+#TERMUX_PKG_SRCURL=https://github.com/rapid7/metasploit-framework/archive/${TERMUX_PKG_VERSION}.tar.gz
+# Depend on coreutils for bin/env
+TERMUX_PKG_DEPENDS="wget, ruby, curl, autoconf, bison, clang, coreutils, curl, findutils, git, libffi-dev, libgmp-dev, libpcap-dev, postgresql-dev, readline-dev, libsqlite-dev, openssl-dev, libtool, libxml2-dev, libxslt-dev, ncurses-dev, pkg-config, postgresql-contrib, wget, make, ruby-dev, libgrpc-dev, termux-tools, ncurses-utils, ncurses, unzip, zip, tar, autoconf "
```

Comment:
do you think that `nmap` should be added to this list?

Reply:
Yeah metasploit-framework uses nmap, ill add it..